config/webpacker.yml

```diff
@@ -0,0 +1,92 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  source_path: app/javascript
+  source_entry_path: packs
+  public_root_path: public
+  public_output_path: packs
+  cache_path: tmp/cache/webpacker
+  webpack_compile_output: true
+
+  # Additional paths webpack should lookup modules
+  # ['app/assets', 'engine/foo/app/assets']
+  resolved_paths: []
```

Comment:
I think you are not using webpack at all. As you can see here, webpack looks for assets in `app/javascript`. You have 2 options here:

1. Move all the existing assets to `app/javascript` (`.js` go in `app/javascript/packs`).
2. Or, add `app/assets` on the list of `resolved_paths`

Reply:
Sorry, I forgot it. I've added the assets path on the list.

Reply:
We also need to modify the `app/javascript/packs/application.js` to include the assets which are going to be served through webpack and move all assets to `app/javascript` as described on the [webpacker documentation](https://github.com/rails/webpacker#usage). Since we are short on time and this is not a major issue, we can postpone this for a later update.
**NOTE:** This applies to all other rails tutorials which @rojasTob updated.

Reply:
Thanks @mrcrow85 for your review. About the last improvement, I will create an issue explaining the last step that it needs to run Webpack. I will create it for each Rails repository.